libs/ledger/ledger-utxo-data/src/main/kotlin/net/corda/ledger/utxo/data/transaction/UtxoLedgerTransactionImpl.kt

```diff
@@ -115,6 +117,16 @@ class UtxoLedgerTransactionImpl(
         return outputContractStates.filterIsInstance(type)
     }
 
+    /*
+    * The null check in the method is to prevent 5.0 from hitting this method since this field doesn't exist in 5.0
```

Comment:
```suggestion
    * The null check in the method is to make 5.0 in-flight transactions caught in upgrades failing more gracefully.
```

If this is correct at all.